src/rt/sections_elf_shared.d

```diff
@@ -352,6 +388,12 @@ else
 // Compiler to runtime interface.
 ///////////////////////////////////////////////////////////////////////////////
 
+version (OSX)
+    private alias ImageHeader = mach_header*;
```

Comment:
Will this work for both 32bit and 64bit? There's a `mach_header_64` type as well.

Reply:
It does work for LDC, apparently for the reason you mentioned (used for `_dyld_get_image_header_containing_address()`, `_dyld_get_image_slide()` and LDC-specific hacky [getSection()](https://github.com/ldc-developers/druntime/blob/ldc/src/rt/mach_utils.d#L31); the latter is available in the `sections_osx_*.d` files here and would need to be moved.]